### ISSUE TITLE
html.option excepts only string not a list of elements 

### ISSUE THREAD

**Issue by abdulhakim-alshanqiti** [lpil/html-lustre-converter#17]
```html
<select >
  <option>1</option>
  <option>2</option>
</select>

```

gets turned into
```rust
html.select(
  [],
  [html.option([], [html.text("1")]), html.option([], [html.text("2")])],
)
```

it should be 

```rust
html.select(
  [],
  [html.option([], "1" ), html.option([], "2" )],
)
```

**Comment by lpil** [comment]
Thank you

**Comment by abdulhakim-alshanqiti** [comment]
thank you for this great tool☝️☝️and this cool language

**Comment by lpil** [comment]
Thank you again! You are very kind!

I will get this fixed soon.

**Comment by giacomocavalieri** [comment]
This was closed by #18!
